### ISSUE TITLE
Notificaciones de chat repetida 

### ISSUE THREAD

**Issue by Pablocalvo321** [onelife-social/support#227]
V2.14.13 

Me ha llegado doble el mismo mensaje de chat 
![IMG_2819](https://github.com/user-attachments/assets/156050bd-0e5f-4e46-88b1-ddc91acb7345)

**Comment by AlbertoJativa** [comment]
Creo que se solucionó

**Comment by jnavarro90** [comment]
Sí, en principio se solucionó en esta tarea: https://github.com/onelife-social/onelife_notifications/issues/190
